src/panels/lovelace/entity-rows/hui-media-player-entity-row.js

```diff
@@ -0,0 +1,115 @@
+import '@polymer/paper-icon-button/paper-icon-button.js';
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+
+import '../components/hui-generic-entity-row.js';
+
+import LocalizeMixin from '../../../mixins/localize-mixin.js';
+
+const SUPPORT_PAUSE = 1;
+const SUPPORT_NEXT_TRACK = 32;
+
+/*
+ * @appliesMixin LocalizeMixin
+ */
+class HuiMediaPlayerEntityRow extends LocalizeMixin(PolymerElement) {
+  static get template() {
+    return html`
+      <style>
+        .controls {
+          white-space: nowrap;
+        } 
+      </style>
+      <hui-generic-entity-row
+        hass="[[hass]]"
+        config="[[_config]]"
+      >
+        <div class="controls">
+          <paper-icon-button 
+            icon="[[_computeControlIcon(_stateObj)]]" 
+            on-click="_playPause" 
+          ></paper-icon-button>
+          <template is="dom-if" if="[[_supportsNext(_stateObj)]]">
+            <paper-icon-button 
+              icon="hass:skip-next" 
+              on-click="_nextTrack" 
+            ></paper-icon-button>
+          </template>
+        </div>
+        
+        <div slot="secondary">
+          [[_computeMediaTitle(_stateObj)]] 
+        </div>
+      </hui-generic-entity-row>
+    `;
+  }
+
+  static get properties() {
+    return {
+      hass: Object,
+      _config: Object,
+      _stateObj: {
+        type: Object,
+        computed: '_computeStateObj(hass.states, _config.entity)'
+      }
+    };
+  }
+
+  _computeStateObj(states, entityId) {
+    return states && entityId in states ? states[entityId] : null;
+  }
+
+  setConfig(config) {
+    if (!config || !config.entity) {
+      throw new Error('Entity not configured.');
+    }
+    this._config = config;
+  }
+
+  _computeControlIcon(stateObj) {
+    if (stateObj.state !== 'playing') {
+      return 'hass:play';
+    }
+
+    return stateObj.attributes.supported_features & SUPPORT_PAUSE ? 'hass:pause' : 'hass:stop';
+  }
+
+  _computeMediaTitle(stateObj) {
+    if (!stateObj) return null;
+
+    let title;
+    switch (stateObj.attributes.media_content_type) {
+      case 'music':
+        title = `${stateObj.attributes.media_artist}: ${stateObj.attributes.media_title}`;
```

Comment:
replace title and break with return

Reply:
what is your preference here: 
1. switch with default case
2. switch with final return outside
3. if-else

Reply:
1 :P

Reply:
Just wanted to make sure 😉